job_executor/model/datastore.py

```diff
@@ -265,84 +268,136 @@ def bump_version(
         Release a new version of the datastore with the pending
         operations in the draft version of the datastore.
         """
-        self._log(job_id, 'initiated')
-        job_service.update_job_status(job_id, 'initiated')
+        self._log(job_id, 'Saving temporary backup')
         local_storage.save_temporary_backup()
-        latest_data_versions = local_storage.get_data_versions(
-            self.latest_version_number
-        )
-        if not self.draft_version.validate_bump_manifesto(bump_manifesto):
-            raise VersioningException(
-                'Invalid Bump: Changes were made to the datastore '
-                'after bump was requested'
+
```

Comment:
This method is a bit long and not easy to read. Refactor?

Reply:
I agree 👍🏻 My hope was the new log messages would make it more readable, but I can try splitting it up into a couple of private methods 💯 Should be very doable